bin/cpu_profiling

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash -e
 
-for i in {1..10}; do
+#Note: use _ instead of i to avoid shellcheck error
```

Comment:
is this because we don't use `i`? I think it's fine to make this change without the note